### PR TITLE
Py3 only

### PR DESCRIPTION
**Pull request by joshmoore**
Late addition to the futurization. I tried to avoid unnecessary imports since Python 2 support has been dropped.

Tag: `0.2.0`

### CODE REVIEW

File: omero_signup/views.py [hunk 1]
```diff
@@ -204,10 +204,11 @@ def _create_user(self, adminc, user, groupid):
         def _convert_unicode(s):
             return str(smart_str(s))
 
-        omeuser = dict((k, _convert_unicode(v)) for (k, v) in user.items())
+        omeuser = dict((k, _convert_unicode(v))
+                       for (k, v) in list(user.items()))
```
Comment (manics) [inline omero_signup/views.py:208]: This change shouldn't be needed

File: omero_signup/views.py [hunk 2]
```diff
@@ -204,10 +204,11 @@ def _create_user(self, adminc, user, groupid):
...
         omeuser['login'] = _convert_unicode(self._get_new_login(adminc, user))
         omeuser['password'] = ''.join(random.choice(
-            string.ascii_letters + string.digits) for n in xrange(12))
+            string.ascii_letters + string.digits) for n in list(range(12)))
```
Comment (manics) [inline omero_signup/views.py]:
```suggestion
            string.ascii_letters + string.digits) for n in range(12))
```

**Comment by joshmoore** [comment]
@manics : travis is now green.

**Comment by manics** [comment]
Seem OK to me. Is the plan to merge, release, test, or test before tagging?

**Comment by joshmoore** [comment]
Unless someone is up for the latter, I'd propose the former.

cc: @mtbc

**Comment by mtbc** [comment]
Go for it, releases are cheap.

**Pull request merged**